crates/ruff_python_formatter/src/lib.rs

```diff
@@ -168,10 +168,9 @@ pub fn format_node<'a>(
 }
 
 /// Public function for generating a printable string of the debug comments.
-pub fn pretty_comments(formatted: &Formatted<PyFormatContext>, source: &str) -> String {
-    let comments = formatted.context().comments();
+pub fn pretty_comments(root: &Mod, comment_ranges: &CommentRanges, source: &str) -> String {
```

Comment:
This is not strictly related to the PR but I ran into situations where I missed to format some comments but was unable to see the comments in the playground because it extracted them from the formatted output... 

This change now allows to see the comments even if the formatting itself fails.